index.html

```diff
@@ -6558,13 +6558,13 @@ <h2 id="defining-a-valid-audioworkletprocessor">
               <ul>
                 <li>Nodes that transform their inputs, and are active only
                 while connected inputs and/or script references exist. Such
-                nodes should return <code>false</code> from
+                nodes SHOULD return <code>false</code> from
```

Comment:
Fair enough.  This is ok.  But I also notice that in the bullet items after this one, there are also some "should" references that look pretty much like this one.  Should these be converted to "SHOULD" instead of "should" too?